src/applications/disability-benefits/view-payments/components/view-payments-lists/ViewPaymentsLists.jsx

```diff
@@ -27,23 +30,103 @@ class ViewPaymentsLists extends Component {
     this.props.getAllPayments();
   }
 
+  buildReturnedPaymentListContent = returnPayments => {
+    // If there are returned payments, set paymentsReturnedTable to a Payments component
+    // If there are NO returned payments, set paymentsReturnedTable to AlertBox
+    const paymentsReturnedTable = {};
+
+    if (returnPayments.length > 0) {
+      // convert date to more friendly format
+      const filteredReturnPayments = reformatReturnPaymentDates(returnPayments);
+
+      paymentsReturnedTable.content = (
+        <Payments
+          tableVersion="returned"
+          fields={paymentsReturnedFields}
+          data={filteredReturnPayments}
+          textContent={paymentsReturnedContent}
+        />
+      );
+    } else {
+      paymentsReturnedTable.content = (
+        <AlertBox
+          content={
+            <p>
+              We can’t find any returned VA payments. If you think this is an
+              error, or if you have questions about your payment history, please
+              call <Telephone contact={CONTACTS.VA_BENEFITS} />.
+            </p>
+          }
+          headline="We don’t have a record of returned payments"
+          status="info"
+          backgroundOnly="true"
+          className="vads-u-background-color--gray-lightest"
+        />
+      );
+    }
+    return paymentsReturnedTable;
+  };
+
+  buildPaymentListContent = payments => {
+    // If there are recieved payments, set paymentsReceivedTable to a Payments component
+    // If there are NO recieved payments, set paymentsReceivedTable to AlertBox
+    const paymentsReceivedTable = {};
```

Comment:
Is there a particular reason this is an object? Instead of assigning the `<Payments />` component to `paymentsReceivedTable.content` could it just be assigned directly to `paymentsReceivedTable` and returned? A react component is returned either way, so I don't see the need to assign either as an object property.